inst/tests/tests.Rraw

```diff
@@ -12354,6 +12354,15 @@ test(1953.4, melt.data.table(DT, id.vars = 'id', measure.vars = 'a'),
 DT = data.table(A=INT(1,3,2,3,2), B=1:5)  # respect groups in 1st column (3's and 2's)
 test(1954, forderv(DT, sort=FALSE, retGrp=TRUE), structure(INT(1,2,4,3,5), starts=1:5, maxgrpn=1L))
 
+for (i in 1:4) {  # Skip should work with all types of newlines #3006
+  eol = eols[i]
+  src = paste(c("A", "B", "...", ",,,,,", "c1,c2,c3", "1,2,3"), collapse="\n")
```

Comment:
Thanks for this fix.  It doesn't look like `eol` is being used in this test.  Is it intended that `collapse=` be passed `eol` rather than `\n` always?  Also, how about `for (eol in c("\n", "\r\n", "\r", "\n\r", "\r\r\n"))` for clarity instead of reusing `eols` from the earlier test (saves the reader scrolling up to find eols).

Reply:
You're right, I wanted to use `eol` for `collapse=` but then forgot.

Reply:
Cannot make `eol` the loop variable, because `i` is also needed to make the test number: `1955 + i*0.1`. However, I can redefine the `eols` variable before the loop to make it easier to understand what is being tested.